src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -156,7 +156,7 @@
             :aria-label="`Copy (${ALT_SHORTCUT_LABEL}+C)`"
             class="code-block-copy-button"
             :copy-tooltip="`Copy (${ALT_SHORTCUT_LABEL}+C)`"
-            data-testid="code-block-copy-button"
+            :data-testid="`code-block-copy-button${id.length ? '-' + id : '' }`"
```

Comment:
```suggestion
            :data-testid="`code-block-copy-button${id ? '-' + id : '' }`"
```

suggestion: no need to check length here

Reply:
Added the changes